File: Data Structures & Algorithms/top-k-elements-in-list/submission-6.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        d = {}
        for i in nums:
            if i in d.keys():
                d[i] += 1
            else:
                d[i] = 1
        
        v = list(d.values())
        ke = list(d.keys())

        l = []
        for i in range(k):
            m = max(v)
            pos = v.index(m)          
            l.append(ke[pos])
            ke.pop(pos)
            v.pop(pos)
        return l

File: Data Structures & Algorithms/jump-game/submission-6.cpp
class Solution {
public:
    bool canJump(vector<int>& nums) {
        int ind = size(nums)-1;
        for (int i = ind-1; i>=0; i--) {
            cout << nums[i] << "\t" << i << "\n";
            if ((i+nums[i]) >= ind)
                ind = i;
        }
        cout << ind;
        if (ind == 0)
            return true;
        return false;
    }
};
